contracts/vaults/RibbonThetaVault.sol

```diff
@@ -158,10 +155,11 @@ contract RibbonThetaVault is DSMath, GnosisAuction, OptionsVaultStorage {
         isPut = _isPut;
 
         // hardcode the initial withdrawal fee
-        instantWithdrawalFee = 0.005 ether;
+        instantWithdrawalFee = 0 ether;
```

Comment:
Leaving this as 0 ether for now, will remove withdrawal fee logic completely in a follow-up PR